policies/trigger-retries.rego

```diff
@@ -0,0 +1,15 @@
+package spacelift
+
+# You can read more about trigger policies here:
+#
+# https://docs.spacelift.io/concepts/policy/trigger-policy
+
+trigger[stack.id] {
+  stack := input.stack
+  input.run.state == "FAILED"
+  input.run.type == "TRACKED"
+  is_null(input.run.triggered_by)
+}
```

Comment:
How does this avoid an infinite loop if the failure is due to a problem with the stack (like a syntax error)?

Reply:
we need to discuss that

Reply:
thanks for reviewing @Nuru 
you expressed exactly the two concerns that I had: 1) the `notrigger` and 2) automatic retries